ArduCopter/mode_guided.cpp

```diff
@@ -43,7 +44,12 @@ struct Guided_Limit {
 bool ModeGuided::init(bool ignore_checks)
 {
     // start in position control mode
-    pos_control_start();
+    vel_control_start();
```

Comment:
This init code looks a bit odd. The velocity controller is started but then it sets position targets and the posvel_update_time_ms.  My guess is that it would be easier and perhaps safer to call set_destination_posvel or set_destination_posvelaccel and provide the stopping point.

Reply:
Removed setting the position. Let the velocity controller bring the aircraft to a stop. That way we can't get overshoot or bounce back.